nyanMigen.py

```diff
@@ -378,6 +378,11 @@ def _convert_generic_with(code, ctx):
 
     @converter
     def _memory_converter(code, ctx):
+        def parse_signal(code):
+            if isinstance(code, Attribute):
+                return (code.attr, code.value.id)
+            elif isinstance(code, Name):
```

Comment:
Using isinstance() is a good choice.  It’s not only faster but also considers inheritance, which is often the desired behavior. Isn't is, @github-4o?